tests/cloudservicesuploadadapter.js

```diff
@@ -60,6 +60,20 @@ describe( 'CloudServicesUploadAdapter', () => {
 				} );
 		} );
 
+		it( 'should not set loader if there is no token', () => {
+			UploadGatewayMock.lastToken = undefined;
+
+			return ClassicTestEditor
+				.create( div, {
+					plugins: [ CloudServicesUploadAdapter ]
+				} )
+				.then( editor => {
+					expect( UploadGatewayMock.lastToken ).to.be.an( 'undefined' );
```

Comment:
Why not `expect( UploadGatewayMock.lastToken ).to.equal( undefined )`?

Reply:
Hmm, that's the same as only `undefined` can give `"undefined"` is a `typeof something`. But I have to admit, that I just reverted this test from the past :D

Reply:
Why not `.to.be.undefined`? :D

Reply:
Ok, I simplified it :D